content/sensu-go/5.15/reference/rbac.md

```diff
@@ -197,6 +197,7 @@ Cluster-wide resources cannot be assigned to a namespace and can only be accesse
 | `cluster`   | Sensu clusters running multiple [Sensu backends][1] |
 | `clusterrolebindings`   | Cluster-wide role assigners  |
 | `clusterroles`   | Cluster-wide permission sets  |
+| `etcd-replicators` | [Mirror RBAC resource changes][40] to follower clusters |
```

Comment:
Need to confirm this is the correct resource type. Or should it be `etcdreplicators` here?

Reply:
This is correct